pom.xml

```diff
@@ -1893,6 +1893,7 @@
                                     -Xep:BoxedPrimitiveConstructor:ERROR \
                                     -Xep:ClassCanBeStatic:ERROR \
                                     -Xep:CompareToZero:ERROR \
+                                    -Xep:DefaultCharset:ERROR \
```

Comment:
This duplicates some rules in Modernizer's `violations.xml`, should they be removed?

Reply:
keep